### PR TITLE
Refactor api routes

### PR DESCRIPTION
**Pull request by Looease**

### CODE REVIEW

File: src/apiRoutes.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import "dotenv/config";
+import {getRoverImages} from "./services/nasaService";
+
+
+const router = express.Router()
+
+
+// define the home page route
+router.get("/rovers/:name/images", async (request :any, response: any) => {
```
Comment (MichaelWalker) [inline src/apiRoutes.ts]: could you remove the `: any`s here?
The 
a) Aren't required - typescript already knows what's happening here
b) Aren't correct - its actually a specific type defined by express - converting it to `any` is effectively removing information.

File: src/app.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ app.use(
 );
 
 //Nunjucks
-const pathToTemplates = "./templates";
+export const pathToTemplates = "./templates";
```
Comment (MichaelWalker) [inline src/app.ts:28]: I don't think we need to export this...

**Review by MichaelWalker — APPROVED** [review]

File: src/apiRoutes.ts (hunk 1 above)
Comment (Looease) [reply src/apiRoutes.ts]: Done

File: src/app.ts (hunk 2 above)
Comment (Looease) [reply src/app.ts:28]: Done

File: src/apiRoutes.ts (hunk 1 above)
Comment (Looease) [reply src/apiRoutes.ts]: The tests fail when we take this out

**Pull request merged**